packages/x-adapter-platform/src/mappers/responses/__tests__/search-response.mapper.spec.ts

```diff
@@ -1,8 +1,11 @@
-import { searchResponseMapper } from '../search-response.mapper';
 import { platformSearchResponse } from '../../../__tests__/__fixtures__/search.response';
+import { PlatformSearchResponse } from '../../../types';
+import { searchResponseMapper } from '../search-response.mapper';
 
 describe('searchResponseMapper tests', () => {
   it('should map the response', () => {
-    expect(searchResponseMapper(platformSearchResponse, {})).toMatchSnapshot();
+    expect(
+      searchResponseMapper(platformSearchResponse as PlatformSearchResponse, {})
```

Comment:
Cast needed to avoid problems with the new `PlatformFacetType`.

Reply:
What do you think of doing this assertion directly on the `__tests__/__fixtures__/search.response` file?